views/layout.pug

```diff
@@ -34,11 +34,9 @@ html
           span.icon-bar
           span.icon-bar
           span.icon-bar
-        a.navbar-brand.scroller.hidden-xs.hidden-sm(href='/')
-          //-if Config.USE_STANFORD_BRAND
-            img(src=Config.ASSET_CDN + '/images/stanford-logo.png', aria-hidden='true')#stanford-logo
+        a.navbar-brand.scroller(href='/')
           span= _("Almond")
-        span.navbar-brand.scroller.hidden-md.hidden-lg(aria-hidden='true')
+        span.navbar-brand.scroller.hidden-xs.hidden-sm.hidden-md.hidden-lg(aria-hidden='true')
```

Comment:
What is this change for?
The purpose of this code is to show the "Almond" on the navbar for desktop screens (md / lg), and show the page title for mobile & tablet (xs, sm).

Reply:
The page title is sometimes too long for mobile screens. Plus I think it doesn't make sense to remove almond branding and show the page title instead on mobile (+ tablet) screens.

Reply:
Well if we make this change we need to make it everywhere - layout.pug is shared across all our pages.

Reply:
Oh, isn't layout.pug the base pug template for the other templates? So making the change here would make the change everywhere already, no? If so, that's what I intended